website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -5436,7 +5436,7 @@ _ = (p | 'Read per user' >> ReadPerUser()
 {{< /highlight >}}
 
 {{< highlight go >}}
-This is not supported yet, see BEAM-10660.
+This is not supported yet, see https://github.com/apache/beam/issues/20510.
```

Comment:
Should these be `[Issue 20510][https://github.com/apache/beam/issues/20510]`?

Reply:
No - this is rendered inside a code block so it would render the full markdown I think

Reply:
Ah good point, thank you